dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.17791.18-r1.ebuild

```diff
@@ -77,6 +77,8 @@ src_configure() {
 	# See https://github.com/intel/intel-graphics-compiler/issues/212
 	append-ldflags -Wl,-z,undefs
 
+	filter-lto
```

Comment:
Please leave a comment explaining why this is being done. e.g. linking to the upstream bug report about this.